src/main/java/org/example/examplefinalProject/controller/TeacherController.java

```diff
@@ -29,4 +32,23 @@ public ModelAndView detail(@PathVariable int teacherId) {
         modelAndView.addObject("teacher", teacherService.findById(teacherId));
         return modelAndView;
     }
+
+    @PostMapping("/createTeacher")
+    public String createTeacher(@RequestParam String teacherName,
+                                @RequestParam String teacherSurname,
+                                @RequestParam String classRoomName) {
+        teacherService.createTeacher(teacherName, teacherSurname, classRoomName);
+        return "redirect:/teacher";
+    }
+
+    @PostMapping("/deleteTeacher")
+    public String deleteTeacher(@RequestParam("teacherId") Integer teacherId, RedirectAttributes redirectAttributes) {
+        try {
+            teacherService.deleteTeacher(teacherId);
+            redirectAttributes.addFlashAttribute("message", "Teacher was delete.");
```

Comment:
what this line of code does?

Reply:
deleteTeacher je metoda která je součástí jiného tasku a toto je message která by měla být na FE ještě není napojeno ze strany FE